app/helpers/grade_entry_forms_helper.rb

```diff
@@ -40,38 +22,22 @@ def set_release_on_grade_entry_students(grade_entry_students, release, errors)
     numGradeEntryStudentsChanged
   end
 
-  # Adds a position to the item's attributes if it doesn't have one
   # Removes items that have empty names (so they don't get updated)
   def update_grade_entry_form_params(attributes)
-
     grade_entry_items =
       params[:grade_entry_form][:grade_entry_items_attributes]
 
-    if grade_entry_items == nil
-      return grade_entry_form_params(attributes)
-    end
-
-    # Find the largest position that has been set
-    max_position = 0
-    grade_entry_items.each_value do |value|
-      next unless value
-      this_position = value[:position]
-      next unless this_position && this_position.to_i > max_position
-      max_position = this_position.to_i
-    end
-
-    # Update the attributes hash
-    max_position += 1
-    grade_entry_items.sort.each do |item|
-      # Items not added don't have a name
-      # Some items are being deleted so don't update those
-      next if item[1][:name] && item[1][:destroy] == 1
-      # If the set position is not valid, update it
-      next if grade_entry_items[item[0]][:position].to_i > 0
-      grade_entry_items[item[0]][:position] = max_position
-      max_position += 1
+    unless grade_entry_items.nil?
+      # Update the attributes hash
+      max_position = 1
+      grade_entry_items.each do |name, item|
```

Comment:
Unused block argument - `name`. If it's necessary, use `_` or `_name` as an argument name to indicate that it won't be used.